lib/faktory_worker/worker_logger.ex

```diff
@@ -25,6 +25,10 @@ defmodule FaktoryWorker.WorkerLogger do
     log_info("Error sending 'FAIL' acknowledgement to faktory", jid, args)
   end
 
+  def log_failed_fetch(error, wid) do
```

Comment:
Can change this to work more like `log_ack` and `log_beat`.  That way we can just add additional clauses in the future if we wanted to add extra logging for fetch without having a `log_fetch` and `log_failed_fetch`.

Something like this

```elixir
def log_fetch(:error, wid, reason) do
  log_info("Failed to fetch job due to '#{reason}' response", wid)
end
```